google/bigtable/v2/bigtable_gapic.yaml

```diff
@@ -0,0 +1,131 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
```

Comment:
sidenote, unrelated to this review - we should get rid of this setting, it doesn't actually impact anything any more.